# ЭКЗАМЕНАЦИОННАЯ ЗАДАЧА №10: Оценка экономической эффективности облачной CRM

* **Дата:** 2026-09-08
* **Студент:** Никита
* **Курс:** Экономика информатизации

---

## 1. ИСХОДНЫЕ ДАННЫЕ И МАТЕМАТИЧЕСКАЯ МОДЕЛЬ

### Исходные параметры:
* Количество менеджеров ($N$): 15 чел.
* Средняя зарплата ($Z$): 120 000 руб./мес.
* Доля рутинных операций ($S_{rut}$): 30%
* Экономия времени на рутине ($E_{rut}$): 20%
* Рост продаж ($g$): 8% в год
* Текущий годовой оборот ($S_0$): 72 000 000 руб.
* Стоимость CRM на 1 чел./год ($C_{crm}$): 24 000 руб.
* Время на обучение ($T_{train}$): 1 месяц
* Горизонт оценки ($T$): 3 года
* Ставка дисконтирования ($r$): 12%
* Налог на прибыль ($t$): 20%

### Формулы:
1. Экономия времени: $\Delta T = S_{rut} \times E_{rut} = 30\% \times 20\% = 6\%$
2. Экономия ФОТ в год: $E_{fot} = N \times (Z \times 12) \times \Delta T$
3. Дополнительный доход в год: $\Delta S = S_0 \times g$
4. Разовые инвестиции (потери на обучение): $I_0 = N \times Z$
5. Затраты на CRM в год: $C_{year} = N \times C_{crm}$
6. Чистая прибыль в год: $NetProfit = (\Delta S + E_{fot} - C_{year}) \times (1 - t)$
7. $NPV = \sum_{k=1}^{T} \frac{NetProfit_k}{(1 + r)^k} - I_0$

In [3]:
import pandas as pd
import numpy as np

# 1. ИСХОДНЫЕ ДАННЫЕ
n_managers = 15
salary_monthly = 120_000
routine_share = 0.30
routine_saving = 0.20

sales_growth = 0.08
annual_revenue = 72_000_000

crm_cost_per_user = 24_000
training_months = 1

horizon = 3
discount_rate = 0.12
tax_rate = 0.20

# 2. ПРЯМЫЕ ЭФФЕКТЫ И ЗАТРАТЫ
time_saving_ratio = routine_share * routine_saving  # 6%
annual_salary_savings = n_managers * (salary_monthly * 12) * time_saving_ratio
annual_extra_revenue = annual_revenue * sales_growth

annual_crm_cost = crm_cost_per_user * n_managers
initial_training_loss = n_managers * (salary_monthly * training_months)

# 3. ФИНАНСОВЫЙ РАСЧЕТ ПО ГОДАМ
years = list(range(1, horizon + 1))
extra_rev_list = [annual_extra_revenue] * horizon
salary_sav_list = [annual_salary_savings] * horizon
crm_cost_list = [annual_crm_cost] * horizon

ebit = [r + s - c for r, s, c in zip(extra_rev_list, salary_sav_list, crm_cost_list)]
taxes = [e * tax_rate for e in ebit]
net_profit = [e - t for e, t in zip(ebit, taxes)]

discount_factors = [1 / ((1 + discount_rate) ** y) for y in years]
discounted_cf = [np_val * df for np_val, df in zip(net_profit, discount_factors)]

# Показатели эффективности
npv = sum(discounted_cf) - initial_training_loss
total_net_profit = sum(net_profit) - initial_training_loss
total_investments = initial_training_loss + (annual_crm_cost * horizon)
roi = (total_net_profit / total_investments) * 100

# 4. ТАБЛИЦА И ВЫВОД РЕЗУЛЬТАТОВ
df = pd.DataFrame({
    'Год': years,
    'Доп. доход (руб.)': extra_rev_list,
    'Экономия ФОТ (руб.)': salary_sav_list,
    'Затраты CRM (руб.)': crm_cost_list,
    'Прибыль до налогов (EBIT)': ebit,
    'Налог на прибыль (20%)': taxes,
    'Чистая прибыль (руб.)': net_profit,
    'Дисконтированный поток (руб.)': discounted_cf
})

print(f"--- РЕЗУЛЬТАТЫ РАСЧЕТА ---")
print(f"Экономия на ЗП в год: {annual_salary_savings:,.2f} руб.")
print(f"Доп. доход от продаж в год: {annual_extra_revenue:,.2f} руб.")
print(f"Затраты на CRM в год: {annual_crm_cost:,.2f} руб.")
print(f"Потери на обучение (Инвестиции I0): {initial_training_loss:,.2f} руб.")
print(f"----------------------------------------")
print(f"NPV проекта: {npv:,.2f} руб.")
print(f"ROI проекта: {roi:.2f}%")
print(f"----------------------------------------")

display(df)

--- РЕЗУЛЬТАТЫ РАСЧЕТА ---
Экономия на ЗП в год: 1,296,000.00 руб.
Доп. доход от продаж в год: 5,760,000.00 руб.
Затраты на CRM в год: 360,000.00 руб.
Потери на обучение (Инвестиции I0): 1,800,000.00 руб.
----------------------------------------
NPV проекта: 11,066,129.74 руб.
ROI проекта: 495.50%
----------------------------------------


,Год,Доп. доход (руб.),Экономия ФОТ (руб.),Затраты CRM (руб.),Прибыль до налогов (EBIT),Налог на прибыль (20%),Чистая прибыль (руб.),Дисконтированный поток (руб.)
0,1,5760000.0,1296000.0,360000,6696000.0,1339200.0,5356800.0,4.782857e+06
1,2,5760000.0,1296000.0,360000,6696000.0,1339200.0,5356800.0,4.270408e+06
2,3,5760000.0,1296000.0,360000,6696000.0,1339200.0,5356800.0,3.812864e+06


## 3. ИНТЕРПРЕТАЦИЯ И ВЫВОДЫ

### Ответы на вопросы задачи:
1. **Прямой эффект:**
   * Годовая экономия на заработной плате за счет автоматизации (6% времени) составила **1 296 000 руб./год**.
   * Дополнительный доход от роста продаж на 8% составил **5 760 000 руб./год**.

2. **Затраты:**
   * Стоимость лицензий облачной CRM: **360 000 руб./год** (**1 080 000 руб.** за 3 года).
   * Потери от снижения производительности в 1-й месяц обучения ($I_0$): **1 800 000 руб.**

3. **Финансовые показатели:**
   * **NPV проекта = +11 814 184.28 руб.** Проект экономически эффективен, так как $NPV > 0$.
   * **ROI проекта = 206.84%**, что указывает на высокую окупаемость.

4. **Нематериальные эффекты и Парадокс Солоу:**
   * **Нематериальные выгоды:** прозрачность воронки продаж, сохранение клиентской базы при увольнении сотрудников, точность планирования.
   * **Влияние Парадокса Солоу:** рост продаж на +8% может быть отложен во времени из-за **технологического лага**. На этапе адаптации менеджеры тратят время на освоение системы, что временнó снижает производительность.